server/src/main/java/com/unosquare/carmigo/dto/GrabDriverDTO.java

```diff
@@ -10,5 +9,5 @@
 
     private String licenseNumber;
 
-    private PlatformUser platformUser;
```

Comment:
This should really be a DTO, not an Entity type.

Reply:
This is weird. This is already a DTO as per last [commit](https://github.com/kleyton-ohare/car-migo/blob/JourneyApi/server/src/main/java/com/unosquare/carmigo/dto/GrabDriverDTO.java) of this file.